go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	google.golang.org/genproto v0.0.0-20170818010345-ee236bd376b0
 	google.golang.org/grpc v1.8.0
 )
+
+go 1.13
```

Comment:
This package supports Go 1.11 in order to support code running on Google App Engine (which not only supports 1.11, but in fact requires 1.11 for App Engine projects that still make use of old App Engine Go packages). In fact, we have code at my work that is still on this runtime, so I need this to work for that older version.

And to make Go 1.11 happy, you need to undo the changes in this file. If you click the link to the red Travis build, you'll see what I mean.

Reply:
Oh, whups. My editor automatically did this. Grumble.

Reply:
Fixed.